### DISCUSSION TITLE
Pricebooks - list price

### DISCUSSION THREAD

**Discussion by ArniSamano** [YetiForceCompany/YetiForceCRM#17422] [General]
Have seens some post yesterday here about pricebooks and also have question. In my case I cannot set "Specify the margin". No changes comes through if I do so. One thing I can see what differs from git.stable is that in mine instance the columns "List price" I can see as default '0' but only one '0', where got.stable have pattern such as 0.00000.
Where to check issue? No logs are generated. My instance is perfectly configured with all necessary php extensions in place.
Help appreciated

**Comment by ArniSamano** [comment]
Answer: basically if we set in user to Truncate trailing zeros, the system doesn't process code related to "Specify the margin".
I consider it as a bug, as user settings unexpectedly causing serious functional issues. Gonna leave this discussion open, as to allow main developer to decide whether fix it or not and than close current discussion.

Best!

**Comment by TonyM15** [comment]
List price field can not be set in PDF by default... would be nice to fix it in core
